.github/workflows/reusable-build-test-release.yml

```diff
@@ -675,7 +698,13 @@ jobs:
           name: package-deployment
           path: build/package/deployment**
         if: ${{ !cancelled() }}
-
+      - name: app-id
```

Comment:
I believe this is not necessary after switching logic of accessing packages from splunkbase to aws s3.

Reply:
Removed